buildpipeline/pipelines.json

```diff
@@ -97,59 +105,96 @@
             "Rid": "linux"
           },
           "ReportingParameters": {
-            "OperatingSystem": "RedHat 7"
+            "OperatingSystem": "RedHat 7",
+            "Type": "build/product/"
           }
         },
         {
           "Name": "DotNet-CoreClr-Trusted-Mac",
           "Parameters": {
             "Rid": "osx.10.12"
+          },
+          "ReportingParameters": {
+            "OperatingSystem": "OSX 10.12",
+            "Type": "build/product/"
           }
         },
         {
           "Name": "DotNet-CoreClr-Trusted-Mac",
           "Parameters": {
             "Rid": "osx",
             "portableBuild": "-portable"
+          },
+          "ReportingParameters": {
+            "OperatingSystem": "RedHat 7",
+            "Type": "build/product/",
+            "SubType": "PortableBuild"
           }
         },
         {
           "Name": "DotNet-CoreClr-Trusted-Windows",
           "Parameters": {
             "Architecture": "x64"
+          },
+          "ReportingParameters": {
+            "OperatingSystem": "Windows",
+            "Type": "build/product/"
           }
         },
         {
           "Name": "DotNet-CoreClr-Trusted-Windows",
           "Parameters": {
             "Architecture": "x64",
             "portableBuild": "-portable"
+          },
+          "ReportingParameters": {
+            "OperatingSystem": "Windows",
+            "Type": "build/product/",
+            "SubType" : "PortableBuild"
           }
         },
         {
           "Name": "DotNet-CoreClr-Trusted-Windows",
           "Parameters": {
             "Architecture": "arm64"
```

Comment:
note if PB_ConfigurationGroup isn't a parameter or Reporting parameter, it won't render due to the view here:
https://github.com/dotnet/core-eng/blob/master/mission-control-config/dotnet/coreclr/viewconfiguration.json#L93
However if it's an existing, better parameter (i.e. ConfigurationGroup sans PB_, that'd be useful.